sanssouci/post_hoc_bounds.py

```diff
@@ -154,44 +155,46 @@ def curve_max_fp(p_values, thr):
         Annals of Statistics, 48(3), 1281-1303.
     """
 
-    #  make sure that p_values and thr are sorted
+    #  make sure that p_values and thresholds are sorted
     if np.linalg.norm(p_values - p_values[np.argsort(p_values)]) > 0.0001:
         p_values = p_values[np.argsort(p_values)]
         print("The input p-values were not sorted -> this is done now")
 
-    if np.linalg.norm(thr - thr[np.argsort(thr)]) > 0.0001:
-        thr = thr[np.argsort(thr)]
-        print("The input thr were not sorted -> this is done now")
+    if np.linalg.norm(thresholds - thresholds[np.argsort(thresholds)]) \
+       > 0.0001:
+        thresholds = thresholds[np.argsort(thresholds)]
+        print("The input 'thresholds' were not sorted -> this is done now")
 
     # do the job
     p = p_values.shape[0]
-    kMax = thr.shape[0]
+    k_max = thresholds.shape[0]
 
-    if kMax < p:
-        thr = np.concatenate((thr, thr[-1] * np.ones(p - kMax)))
-        kMax = thr.shape[0]
+    if k_max < p:
+        thresholds = np.concatenate((thresholds, thresholds[-1] *
+                                    np.ones(p - k_max)))
+        k_max = thresholds.shape[0]
 
-    K = np.ones(p) * (kMax)
+    K = np.ones(p) * (k_max)
     # K[i] = number of k/ T[i] <= s[k] = BB in 'Mein2006'
-    Z = np.ones(kMax) * (p)
+    Z = np.ones(k_max) * (p)
     # Z[k] = number of i/ T[i] >  s[k] = cardinal of R_k
     # 'K' and 'Z' are initialized to their largest possible value,
-    #       ie 'p' and 'kMax', respectively
+    #       ie 'p' and 'k_max', respectively
 
     kk = 0
     ii = 0
 
-    while (kk < kMax) and (ii < p):
-        if thr[kk] >= p_values[ii]:
+    while (kk < k_max) and (ii < p):
+        if thresholds[kk] >= p_values[ii]:
             K[ii] = kk
             ii += 1
         else:
             Z[kk] = ii
             kk += 1
 
-    Vbar = np.zeros(p)
+    max_fp_ = np.zeros(p)
     ww = np.where(K > 0)[0]
```

Comment:
this `ww` variable is probably not useful, and can simply be replaced by `K > 0` wherever it appears.